### PR TITLE
Fix site new on header

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/site/site_new/site_new_controller.js [hunk 1]
```diff
@@ -41,11 +41,14 @@ function SiteNewController($scope, $rootScope, $state, $mdToast, API, AuthServic
...
       function (data, status, headers, config) {
         form.loading = false;
-        // Update sites
-        $scope.user.sites.push(data);
 
         // Update current user's data
-        AuthService.updateAuthentication($scope.user);
+        AuthService.updateAuth($scope.user);
+
+        //
```
Comment (Alir3z4) [inline app/site/site_new/site_new_controller.js]: What's this ? Just remove the empty comment and let be only one blank line between them.

File: app/site/site_new/site_new_controller.js [hunk 2]
```diff
@@ -41,11 +41,14 @@ function SiteNewController($scope, $rootScope, $state, $mdToast, API, AuthServic
...
+        $rootScope.$broadcast('getSite', data);
+
+        console.log(data);
```
Comment (Alir3z4) [inline app/site/site_new/site_new_controller.js]: Remove console.log if you don't need it.

File: app/header/header_controller.js [hunk 3]
```diff
@@ -46,4 +50,4 @@ function HeaderController($scope, $state, $stateParams, AuthService) {
 };
 
 app.controller("HeaderController", HeaderController);
-HeaderController.$inject = ['$scope', '$state', '$stateParams', 'AuthService'];
+HeaderController.$inject = ['$scope', '$rootScope', '$state', '$stateParams', 'AuthService'];
```
Comment (Alir3z4) [inline app/header/header_controller.js]: Separate them by line.

```
HeaderController.$inject = [
  '$scope',
  '$rootScope', 
  '$state',
  '$stateParams', 
  'AuthService'
];
```

File: app/site/site_new/site_new_controller.js (hunk 1 above)
Comment (ArsalanSavand) [inline app/site/site_new/site_new_controller.js]: I didn't see it, OK I will

**Comment by ArsalanSavand** [comment]
All Done!

**Pull request merged**